src/sagemaker/jumpstart/hub/hub.py

```diff
@@ -68,7 +67,9 @@ def __init__(
         self,
         hub_name: str,
         bucket_name: Optional[str] = None,
-        sagemaker_session: Optional[Session] = DEFAULT_JUMPSTART_SAGEMAKER_SESSION,
+        sagemaker_session: Optional[
```

Comment:
i wouldn't set this as a default argument since this function will get invoked whenever the module is imported, which may cause slow latency or errors on some systems. can you set in constructor body instead?

Reply:
thanks changed it in the new revision